skills/lark-doc/references/lark-doc-media-insert.md

```
@@ -100,7 +100,91 @@ lark-cli docs +media-insert --doc doxcnXXX --from-clipboard --align center --cap
 > [!CAUTION]
 > 这是**写入操作**（会修改文档内容）—— 执行前必须确认用户意图。
 
+## 插入图片到容器 block
+
+`+media-insert` 默认将图片插入到文档顶层（末尾或 `--selection-with-ellipsis` 匹配的顶层 block 附近），无法直接放入表格单元格、callout、grid column 等容器内部。通过"上传后移动"两步工作流可实现：
+
+### 工作流
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)